client/views/view_units.cpp

```diff
@@ -144,16 +151,16 @@ void units_view::update_view()
 
     ui.units_widget->insertRow(max_row);
     for (j = 0; j < 7; j++) {
-      item = new QTableWidgetItem;
+      QTableWidgetItem *item = new QTableWidgetItem;
       switch (j) {
       case 0:
         // Unit type image sprite and name
         if (sprite != nullptr) {
           item->setData(Qt::DecorationRole, sprite->scaledToHeight(h));
         }
         item->setData(Qt::UserRole, id);
-        item->setTextAlignment(Qt::AlignLeft);
         item->setText(utype_name_translation(putype));
+        item->setTextAlignment(Qt::AlignLeft | Qt::TextAlignmentRole);
```

Comment:
Surprised by the `TextAlignmentRole`, was it that or swapping `setText` and `setTextAlignment` that made text left aligned?

Reply:
I'm not sure it works. Will need to come back at it when we change it to a `QTreeView`. I'm going to leave as is for now.